### DISCUSSION TITLE
<title> Can't use Copilot even after receiving the Email

### DISCUSSION THREAD

**Discussion by Abrygzy** [community/community#44052] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I am a student verified by Github. After receiving the renewal email, I still do not have the access to copilot. 
I check the previous discussions, most people are able to use copilot after getting that email. Is there anything I can do?
<img width="554" alt="image" src="https://user-images.githubusercontent.com/53932200/211970587-a64c5718-6cb8-47b1-9081-685456c8daf8.png">
<img width="805" alt="image" src="https://user-images.githubusercontent.com/53932200/211970626-2d7036f5-b291-412b-a411-97fe8c4ab971.png">

**Comment by huamurui** [comment]
Just met the same, too.
![image](https://user-images.githubusercontent.com/60086749/211971359-ecd559dc-ecf5-4286-bd39-fafd3e3f98e9.png)

It worked for a while after the email, but now I can't access it again.

**Comment by Akash1134** [comment, marked as answer]
Hi @Abrygzy ,

Thank you for taking the time to report and we apologize for the temporary technical issue on our end that may have caused authentication failures. If you are a verified student please retry logging in to your IDE plugin, and Copilot should work as expected.

If you run into any further issues, please reply back in detail with any useful screenshots

**Reply by Abrygzy** [reply]
Thank you for your kind reply! My Copilot now works well.
